src/Styleguide/Components/EntityHeader.tsx

```diff
@@ -20,14 +20,13 @@ export const EntityHeader: SFC<EntityHeaderProps> = ({
   meta,
   FollowButton,
 }) => {
-  const handleContainerClick = () => {
-    if (href) {
-      location.href = href
-    }
-  }
+  const ContainerComponent = href ? FlexLink : Flex
+  const containerProps = href
```

Comment:
The container could either be `<Flex>` or a `<FlexLink>` (which is built on top of `<Link>`) and they accept a different set of props, otherwise we may pass in `href` to `<Flex>` that could be questionable.

Reply:
👍